ci/ci/envoy.py

```diff
@@ -172,13 +172,15 @@ def auth_check_exemption() -> dict:
     }
 
 
-def rate_limit_config() -> dict:
+def rate_limit_config(service: str, namespace: str) -> dict:
+    max_rps = 60 if service == 'batch-driver' and namespace == 'default' else 200
```

Comment:
Shouldn't we rate limit batch-driver in every namespace? Differential rate limiting by namespace seems like it will make understanding test speed and test slowness harder.

Reply:
fair point